index.html

```diff
@@ -854,13 +854,13 @@ <h2>Examples</h2>
           thing.addProperty({
             name: "temperature",
             value: "0",
-            type: "number",
+            schema: "number",
```

Comment:
must be `schema: "{ \"type\": \"number\" }`

Reply:
Thanks! Fixed.